master/buildbot/process/buildstep.py

```diff
@@ -887,6 +899,7 @@ def __init__(self, logfiles={}, lazylogfiles=False, log_eval_func=None,
         if log_eval_func and not callable(log_eval_func):
             config.error(
                 "the 'log_eval_func' paramater must be a callable")
+        self.urls = urls
```

Comment:
Could we add a check type of the parameter and throw a configuration error if we don't get the one we expect?